<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="qsg-PC" name="requirementGroup.CreatingAndManagingUsers.CreateUser" tests="3" failures="1" timestamp="6 Oct 2015 21:05:15 GMT" time="1429.313" errors="1">
  <testcase name="testSmoke68391" time="364.657" classname="requirementGroup.CreatingAndManagingUsers.CreateUser"/>
  <testcase name="testHappyDay68200" time="461.161" classname="requirementGroup.CreatingAndManagingUsers.CreateUser">
    <failure type="org.openqa.selenium.NoSuchElementException" message="Unable to find element with xpath == //div[@class=&#039;footer&#039;]/div[@class=&#039;right&#039;][text()=&#039;Report Created: 06 Oct 2015 09:31 CDT&#039;] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 60.05 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.46.0&#039;, revision: &#039;61506a4624b13675f24581e453592342b7485d71&#039;, time: &#039;2015-06-04 10:22:50&#039;
System info: host: &#039;qsg-PC&#039;, ip: &#039;192.168.27.31&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;x86&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.7.0_05&#039;
*** Element info: {Using=xpath, value=//div[@class=&#039;footer&#039;]/div[@class=&#039;right&#039;][text()=&#039;Report Created: 06 Oct 2015 09:31 CDT&#039;]}
Session ID: 5cf11a88-865c-48e2-b798-15f831bc021f
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{platform=WINDOWS, javascriptEnabled=true, elementScrollBehavior=0, enablePersistentHover=true, ignoreZoomSetting=false, ie.ensureCleanSession=false, browserName=internet explorer, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss, version=9, ie.usePerProcessProxy=false, cssSelectorsEnabled=true, ignoreProtectedModeSettings=false, requireWindowFocus=false, initialBrowserUrl=http://localhost:44881/, handlesAlerts=true, ie.forceCreateProcessApi=false, nativeEvents=true, browserAttachTimeout=0, ie.browserCommandLineSwitches=, takesScreenshot=true}]">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to find element with xpath == //div[@class='footer']/div[@class='right'][text()='Report Created: 06 Oct 2015 09:31 CDT'] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 60.05 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.46.0', revision: '61506a4624b13675f24581e453592342b7485d71', time: '2015-06-04 10:22:50'
System info: host: 'qsg-PC', ip: '192.168.27.31', os.name: 'Windows 7', os.arch: 'x86', os.version: '6.1', java.version: '1.7.0_05'
*** Element info: {Using=xpath, value=//div[@class='footer']/div[@class='right'][text()='Report Created: 06 Oct 2015 09:31 CDT']}
Session ID: 5cf11a88-865c-48e2-b798-15f831bc021f
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{platform=WINDOWS, javascriptEnabled=true, elementScrollBehavior=0, enablePersistentHover=true, ignoreZoomSetting=false, ie.ensureCleanSession=false, browserName=internet explorer, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss, version=9, ie.usePerProcessProxy=false, cssSelectorsEnabled=true, ignoreProtectedModeSettings=false, requireWindowFocus=false, initialBrowserUrl=http://localhost:44881/, handlesAlerts=true, ie.forceCreateProcessApi=false, nativeEvents=true, browserAttachTimeout=0, ie.browserCommandLineSwitches=, takesScreenshot=true}]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:204)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:156)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:605)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:358)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:455)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:358)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:350)
	at qaframework.lib.webElements.WebElements.getOne(WebElements.java:34)
	at lib.page.Report.verifyFooterHTMLMonthlyAssesmentReport(Report.java:1690)
	at requirementGroup.CreatingAndManagingUsers.CreateUser.testHappyDay68200(CreateUser.java:666)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- testHappyDay68200 -->
  <testcase name="testHappyDay68201" time="603.495" classname="requirementGroup.CreatingAndManagingUsers.CreateUser">
    <error type="java.lang.AssertionError" message="expected [Event Detail Report (Step 1 of 2)] but found [Event Detail Report (Step 2 of 2)]">
      <![CDATA[java.lang.AssertionError: expected [Event Detail Report (Step 1 of 2)] but found [Event Detail Report (Step 2 of 2)]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at lib.page.Report.verEventDetailReportPge2IsDisplayed(Report.java:527)
	at requirementGroup.CreatingAndManagingUsers.CreateUser.testHappyDay68201(CreateUser.java:361)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- testHappyDay68201 -->
</testsuite> <!-- requirementGroup.CreatingAndManagingUsers.CreateUser -->
